content/sensu-go/6.0/sensuctl/back-up-recover.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "Back up and recover resources with sensuctl"
+linkTitle: "Back Up and Recover Resources"
+description: "Use the sensuctl command line tool to back up and recover your Sensu resources. Read this page to learn how to use sensuctl dump commands for backup and recovery."
+weight: 20
+version: "6.0"
+product: "Sensu Go"
+platformContent: false 
+menu:
+  sensu-go-6.0:
+    parent: sensuctl
+---
+
+The `sensuctl dump` command allows you to export your [resources][6] to standard out (STDOUT) or to a file.
+You can export all of your resources or a subset of them based on a list of resource types.
+The `dump` command supports exporting in `wrapped-json` and `yaml`.
+
+For example, to export all resources to a file named `my-resources.yaml` in `yaml` format:
+
+{{< code shell >}}
+sensuctl dump all --format yaml --file my-resources.yaml
+{{< /code >}}
+
+To export only checks to STDOUT in `yaml` format:
+
+{{< code shell >}}
+sensuctl dump check --format yaml
+{{< /code >}}
+
+To export only handlers and filters to a file named `my-handlers-and-filters.yaml` in `yaml` format:
+
+{{< code shell >}}
+sensuctl dump handler,filter --format yaml --file my-handlers-and-filters.yaml
+{{< /code >}}
+
+After you use `sensuctl dump` to back up your Sensu resources, you can [restore][3] them later with [`sensuctl create`][1].
+This page explains how to back up your resources for two common use cases: before a Sensu version upgrade and to populate new namespaces with existing resources.
+
+## Back up before a Sensu version upgrade
+
+You should create a backup before you [upgrade][4] to a new version of Sensu.
+Here's the step-by-step process:
+
+1. Create a backup folder.
+
+   {{< code shell >}}
+mkdir backup
+{{< /code >}}
+   
+2. Create a namespaced backup of the entire cluster, except entities, events, and [role-based access control (RBAC)][2] resources.
+   
+   {{< code shell >}}
+sensuctl dump all \
+--omit entities,events,apikeys,users,roles,rolebindings,clusterroles,clusterrolebindings \
+--format yaml > backup/config.yaml
+{{< /code >}}
+   
+3. Export your [RBAC][2] resources, except API keys and users.
+   
+   {{< code shell >}}
+sensuctl dump roles,rolebindings,clusterroles,clusterrolebindings
+--format yaml > backup/system-rbac.yaml
+{{< /code >}}
+
+4. Export your API keys and users resources.
+   
+   {{< code shell >}}
+sensuctl dump apikeys,users
+--format yaml > backup/cannotrestore.yaml
```

Comment:
I kind of want to write this to some other path, but I'm not sure what to suggest. Someone who follows this instruction and then uses `sensuctl create -r -f backup/` from the section below to restore the backup might experience some confusion.

Reply:
How about [another] note that explains they might want to save it somewhere besides `backup` b/c they can't actually restore it from there?